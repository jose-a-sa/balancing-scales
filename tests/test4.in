#ScaleName,Left,Right
A1,B1,B2
B1,3,B3
B2,B4,7
B3,5,B5
B4,6,B6
B5,2,4
B6,1,1