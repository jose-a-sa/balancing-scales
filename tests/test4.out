A1,0,12
B1,16,0
B2,0,6
B3,4,0
B4,0,3
B5,2,0
B6,0,0
